eml_parser/indicators.py

```diff
@@ -40,3 +45,27 @@ def __hash__(self):
                 self.sha256,
             )
         )
+
+    @staticmethod
+    def parse_auth_results_header(headers: list) -> dict:
+        value = ""
+        auth_types = {}
+
+        if headers:
+            for header in headers:
+                name = header.get("name")
+                if name == "Authentication-Results":
```

Comment:
Are we guaranteed to only have one "Authentication-Results" named header? If so, we should put a `break` on line 59

Reply:
If not, we are losing information in the `value` variable when another loop starts on line 55. But pretty sure we are assuming there is only one here

Reply:
Yes, there is only one Authentication-Results header, so I added a `break` statement in this loop.